src/shared/components/DateRangeDropdown/DateRangeDropdown.tsx

```diff
@@ -83,8 +87,9 @@ const DateRangeDropdown: FunctionComponent<DateRangeDropdownProps> = ({
 	 * Called when the user switches between "year" range and "full date" range controls
 	 * @param type
 	 */
-	const dateTypeChanged = (type: 'year' | 'date') => {
+	const dateTypeChanged = (type: DateType) => {
```

Comment:
ik maak meestal maar dedicated types als het type op meer dan 1 plaats gebruikt wordt. Anders maakt het de code minder leesbaar.

Reply:
I'll keep it in mind for the future :)